javascript/src/components/dashboard/Dashboard.jsx

```diff
@@ -0,0 +1,60 @@
+/* global jsRoutes */
+
+const React = require('react');
+const DropdownButton = require('react-bootstrap').DropdownButton;
+const MenuItem = require('react-bootstrap').MenuItem;
+
+const EditDashboardModalTrigger = require('./EditDashboardModalTrigger');
+const PermissionsMixin = require('../../util/PermissionsMixin');
+
+const DashboardStore = require('../../stores/dashboard/DashboardStore');
+
+const Dashboard = React.createClass({
+  mixins: [PermissionsMixin],
```

Comment:
Could you please add propTypes to the component? This way components are way easier to reuse.